erpnext/patches/v17/add_sales_order_custom_fields_001.py

```
@@ -0,0 +1,79 @@
+        frappe.db.commit()
+        
+        print("Custom fields created successfully")
+        print("   - custom_order_priority")
+        print("   - custom_internal_notes")
+        print("   - custom_approval_required")
+        print("   - custom_approved_by")
+        print("   - custom_approval_date")
+        print("="*60 + "\n")
+        
+    except Exception as e:
+        print(f"Error: {str(e)}")
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)